src/main/java/com/rakovets/course/java/core/practice/arrays/Task01.java

```diff
@@ -34,7 +36,12 @@ public static void main(String[] args) {
     static double getAverageMark(int[] marks) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0.0;
+        int sum = 0;
+        for (int mark: marks) {
+            sum += mark;
+        }
+        double averageMark = (double) sum / marks.length;
+       return NumberUtil.roundValueToTwoDigitsForMantissa(averageMark);
```

Comment:
"Violation: There aren't correct indentations for inner code
Solution: Add correct indentations for inner code"